pkg/instance/instance.go

```diff
@@ -61,6 +62,8 @@ func NewEnv(cfg *mgrconfig.Config) (Env, error) {
 	return env, nil
 }
 
+var incompleteBuildRe = regexp.MustCompile(`Executor will not be built`)
```

Comment:
This will be very fragile. Tomorrow somebody changes that text in the Makefile, no test will fail, even syz-ci's won't fail. This feature will just silently stop working.
There should be a more reliable way to do this.

Reply:
Actually, we can turn that message in Makefile into a build process failure (so that no 0 exit code is returned). It looks reasonable to me -- if we cannot build syz-execution, but it was requested, well, that's an error.

And, as the code modified in this PR is being used also to compile older syzkallers, we can still look for that string in the output. But we'll be sure that it won't break -- it's present in all past versions and not needed for the future versions.

What do you think?